Makefile

```diff
@@ -729,7 +729,8 @@ version: $(VERSRC)
 $(VERSRC): Makefile
 	VERSION=$(VERSION) $(MAKE) -f version.mk setver
 	# Update api module path, but don't fail on error.
-	$(MAKE) update-api-import-path || true
+	# Disabled for v9 release.
+	# $(MAKE) update-api-import-path || true
```

Comment:
Actually, can we just remove this instead of commenting out? Or leave an appropriate TODO item something like "TODO: Enable automatic updating of API import paths using update-api-import-path target once agreed upon the solution".